tests/Doctrine/Tests/OrmFunctionalTestCase.php

```diff
@@ -946,4 +950,60 @@ final protected function getLastLoggedQuery(int $index = 0): array
 
         return $lastQuery;
     }
+
+    public function getLimitSQLByPlatform($limit, AbstractPlatform $platform): string
```

Comment:
There's already an API for building a limit query. Why is another one needed?

Reply:
I didn't make an API. Yes, I know this functionality exists. This was necessary for the correct construction of native queries.

Reply:
> I didn't make an API.

The question isn't who made this API. The question is, is it worth butchering the already bad test suite just in order to have the build pass, and what value this new code provides.

Reply:
good question

Reply:
The new code only gives a well-formed SQL query